src/features/visualization/d3-viz/setup-annotations.js

```diff
@@ -39,8 +39,11 @@ const setupAnnotations = ({packedData, annotationRoot}) =>{
   .append('text')
     .classed('svg-icon', true)
     .classed('nag', true)
-	 .style('font-size', (d) => (3 * d.height * fontScale) + "%")
-    .text('\uf06a');//font-awesome
+	 .style('font-size', (d) => (4 * d.height * fontScale) + "%")
+	 .text('\uf05a');//font-awesome information icon circle
```

Comment:
Please revert this change to the exclamation point. The info icon isn't considered urgent enough

Reply:
Reverted, but I did increase the icon size